go/cmd/vtctld/schema.go

```diff
@@ -36,7 +36,9 @@ var (
 	schemaChangeController    = flag.String("schema_change_controller", "", "schema change controller is responsible for finding schema changes and responding to schema change events")
 	schemaChangeCheckInterval = flag.Int("schema_change_check_interval", 60, "this value decides how often we check schema change dir, in seconds")
 	schemaChangeUser          = flag.String("schema_change_user", "", "The user who submits this schema change.")
-	schemaChangeSlaveTimeout  = flag.Duration("schema_change_slave_timeout", 10*time.Second, "how long to wait for replicas to receive the schema change")
+	// for backwards compatibility
+	deprecatedTimeout           = flag.Duration("schema_change_slave_timeout", 10*time.Second, "deprecated, use schema_change_replicas_timeout instead")
+	schemaChangeReplicasTimeout = flag.Duration("schema_change_replicas_timeout", *deprecatedTimeout, "how long to wait for replicas to receive the schema change")
```

Comment:
just wondering about singular (`slave`) vs plural (`replicas`)?

Reply:
We decided to use `replicas` in the flag name to be consistent with the help text. The old flag name was not.

Reply:
This may not have the intended effect. I think the `schemaChangeReplicasTimeout` will get set to the preset `deprecatedTimeout`, which would be 10s. Following this, if users specified `schema_change_slave_timeout` on the command line, then only deprecatedTimeout will be changed. But `schemaChangeReplicasTimeout` will be unaffected.

Reply:
good catch, I'll figure out a way to fix it.